src/commands/blobs/blobs-delete.ts

```diff
@@ -1,21 +1,35 @@
 import { getStore } from '@netlify/blobs'
 
-import { chalk, error as printError } from '../../utils/command-helpers.js'
+import { chalk, error as printError, log } from '../../utils/command-helpers.js'
+import { promptBlobDelete } from '../../utils/prompts/blob-delete-prompts.js'
 
 /**
  * The blobs:delete command
  */
 export const blobsDelete = async (storeName: string, key: string, _options: Record<string, unknown>, command: any) => {
+  // Prevents prompts from blocking scripted commands
+  if (command.scriptedCommand) {
+    _options.force = true
+  }
+
```

Comment:
Can we do this top level? Ie can we just add it to every option for every command at when we're instantiating the command rather than doing it for every individual command?

Reply:
We decided on this approach as it's more precise (only changes behavior on the commands we added prompting to). We can update to `--force` all commands within CI/non-interactive shell.